cypress/utils/apps/data_explorer/commands.js

```diff
@@ -125,33 +125,49 @@ Cypress.Commands.add(
   }
 );
 
-Cypress.Commands.add('saveQuery', (name, description) => {
+Cypress.Commands.add('saveQueryOldUI', (name, description = ' ') => {
   cy.whenTestIdNotFound('saved-query-management-popover', () => {
     cy.getElementByTestId('saved-query-management-popover-button').click();
   });
   cy.getElementByTestId('saved-query-management-save-button').click();
 
   cy.getElementByTestId('saveQueryFormTitle').type(name);
   cy.getElementByTestId('saveQueryFormDescription').type(description);
+
+  cy.getElementByTestId('savedQueryFormSaveButton').click({ force: true });
```

Comment:
why is `force` needed? Typically, this is only needed if the element is hidden by some other element in the UI (like a popover). if so, then it is valid that it is needed but a comment should be there signifying why it's needed. Else we can just get rid of `force: true`

Reply:
I think this is actually left over from some testing. Will reverify and check.

Reply:
Confirmed. Removing.